script/TestnetScript.s.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.25;
+
+import {IERC20} from '@openzeppelin/contracts/token/ERC20/IERC20.sol';
+import {IBPool} from 'contracts/BPool.sol';
+import {IBFactory} from 'interfaces/IBFactory.sol';
+import {IFaucet} from 'interfaces/IFaucet.sol';
+
+import {Script} from 'forge-std/Script.sol';
+import {Params} from 'script/Params.s.sol';
+
+contract TestnetScript is Script, Params {
+  /// @notice BFactory contract deployment address
+  IBFactory public bFactory = IBFactory(address(0x25E497A2930788e0ecb97D0aA3E1274793F83A91));
```

Comment:
hmmm tbh this file was also used to deploy a pool on testnet to start the graph, is not particularly required, we could also remove the feature since we're not storing the deployed addresses in solidity style, else we should create a `Deployments.s.sol` file and store deployed addresses to script into, i think i incline for that one

Reply:
why not receive the bfactory address as a parameter to `run()`? you just had to edit the source file to configure a new pool

Reply:
would make scripting more difficult imo, and in this way we keep a solidity registry of the contracts, may not be a super feature here, but on protocols with multiple contracts is quite handy